pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/plan/pipeline/PipelineBreakerExecutorTest.java

```diff
@@ -101,6 +104,15 @@ public void tearDownMethod()
     _mocks.close();
   }
 
+  @VisibleForTesting
```

Comment:
Do we need the `@VisibleForTesting` annotation in a Test class? I'm guessing you forgot to remove it after copy-paste it from `PipelineBreakerExecutor`.

Reply:
I forgot to remove it :laughing: